optimum/intel/openvino/modeling_sentence_transformers.py

```diff
@@ -0,0 +1,96 @@
+from types import MethodType
+from typing import Dict, List, Tuple, Union
+
+import numpy as np
+import torch
+from sentence_transformers import SentenceTransformer
+from transformers.file_utils import add_start_docstrings
+from transformers import AutoTokenizer
+
+from .modeling import MODEL_START_DOCSTRING, OVModel
+
+
+@add_start_docstrings(
+    """
+    OpenVINO Model for feature extraction tasks for Sentence Transformers.
+    """,
+    MODEL_START_DOCSTRING,
+)
+class OVModelForSTFeatureExtraction(OVModel):
```

Comment:
model class naming is unclear, maybe better name it as `OVModelForSentenceTransformer`?